collect_app/src/main/java/org/odk/collect/android/preferences/BaseAdminPreferencesFragment.java

```diff
@@ -0,0 +1,38 @@
+package org.odk.collect.android.preferences;
+
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+
+import org.jetbrains.annotations.NotNull;
+import org.odk.collect.android.preferences.stores.AdminPreferencesDataStore;
+
+import javax.inject.Inject;
+
+public abstract class BaseAdminPreferencesFragment extends BasePreferenceFragment {
+    @Inject
+    AdminPreferencesDataStore adminPreferencesDataStore;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        getPreferenceManager().setPreferenceDataStore(adminPreferencesDataStore);
```

Comment:
If we did have one `PreferenceDataStore` implementation we could just inject the `PreferenceDataSourceProvider` here instead of the specific implementation.

Reply:
It's fixed now I inject `PreferencesDataStore`